Master-OB-OpenAPI.json

```diff
@@ -15165,6 +15165,118 @@
         "Curtailed": {
           "label": "Curtailed",
           "description": ""
+        },
+        "Emergency": {
+          "label": "Emergency",
+          "description": ""
+        },
+        "Environment": {
+          "label": "Environment",
+          "description": ""
+        },
+        "Forced": {
+          "label": "Forced",
+          "description": ""
+        },
+        "Grid": {
+          "label": "Grid",
+          "description": ""
+        },
+        "Islanded": {
+          "label": "Islanded",
+          "description": ""
+        },
+        "Shutdown": {
+          "label": "Shutdown",
+          "description": ""
+        },
+        "Standby": {
+          "label": "Standby",
+          "description": ""
+        },
+        "Operational": {
+          "label": "Operational",
+          "description": ""
+        },
+        "UnderMaintenance": {
+          "label": "Under Maintenance",
+          "description": ""
+        },
+        "CommunicationFailure": {
+          "label": "Communication Failure",
+          "description": ""
+        },
+        "Decommissioned": {
+          "label": "Decommissioned",
+          "description": ""
+        },
+        "InformationUnavailable": {
+          "label": "Information Unavailable",
+          "description": ""
+        },
+        "InformationAvailable": {
+          "label": "Information Available",
+          "description": ""
+        },
+        "Nonoperative": {
+          "label": "Nonoperative",
+          "description": ""
+        },
+        "Operative": {
```

Comment:
Let's remove "Operational" in favor of "Operative" since IEC 63019 uses "Operative"